src/index.ts

```diff
@@ -40,6 +40,52 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('hasOperationSupportingMultipart', function(value: CodegenOperation | CodegenOperation[]): boolean {
+			if (isCodegenOperation(value)) {
+				return value.requestBody?.defaultContent.mediaType.mimeType === 'multipart/form-data'
+			}
+
+			return containsMultipartOperation(value)
+		})
+
+		hbs.registerHelper('isMultipartSchema', function(value: CodegenSchema): boolean {
+			return value.originalName === 'multipart/form-data'
```

Comment:
I don't like relying on the name... so I guess we need to have a way to see that this is a special schema (maybe a purpose) _and_ that it was generated for a particular mime-type encoding.

Reply:
I've implemented a potential solution: https://github.com/karlvr/openapi-generator-plus/pull/56